src/app/components/Sidebar/components/ListCategories.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useMemo, useContext } from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import cn from 'classnames';
+
+import { getCategories } from '~utils/queries';
+import { GlobalContext } from '~context/GlobalProvider';
+import { actionCreators } from '~context/GlobalProvider/actions';
+
+import { Categories } from '../interface';
+import styles from '../styles.module.scss';
+
+function ListCategories() {
```

Comment:
Mmmm, I would be careful with this. In #42 there's another implementation so, we might have a merge problem 😝